### DISCUSSION TITLE
Unstable BLE Connection

### DISCUSSION THREAD

**Discussion by greno** [Seeed-Studio/wiki-documents#55] [Q&A]
I'm having an issue keeping a good BLE connection between Windows (using Chrome and Web Bluetooth) and the XIAO ESP32C3.  The ESP32C3 is listed when scanning for devices and I can connect to it, but it immediately disconnects before I can read a characteristic.  I don't have the issue using an old Hiletgo ESP32 dev module.  And yes, I do have the antenna connected.

My setup - using Arduino IDI (2.0.4) with ESP library version 2.0.6.  I tried using 2.0.7, but had problems with incompatibilities with IDE.
Also using the FastLED library for controlling a string of LEDs.  Even with the LEDs disconnected, I still have the same issue.

Any suggestions?

**Comment by MatthewJeffson** [comment]
Hi, the discussion function here for the wiki is just come online and there are few people here. Hope that there will be other contributor s can help you! And you can always ask them on our [forum](https://forum.seeedstudio.com/)!

**Reply by greno** [reply]
I also posted on the esp32.com forum but haven't gotten a response.
